.changeset/slick-pillows-cheer.md

```diff
@@ -0,0 +1,17 @@
+---
+"fluid-framework": minor
+"@fluidframework/tree": minor
+"@fluid-experimental/tree-react-api": minor
+"__section": tree
+---
+Added APIs for tracking observations of SharedTree content for automatic invalidation
```

Comment:
An example usage in the changeset would be good 🙂

Reply:
Added.